package.json

```diff
@@ -69,7 +69,7 @@
 		"rollup-plugin-version-injector": "^1.3.3",
 		"ts-jest": "^28.0.5",
 		"typedoc": "^0.22.18",
-		"typedoc-plugin-djs-links": "^1.1.0",
+		"typedoc-plugin-djs-links": "^1.0.4",
 		"typedoc-plugin-mdn-links": "^1.0.6",
```

Comment:
This was because the typefoc isn't found on yarn registery.

<https://registry.yarnpkg.com/typedoc-plugin-djs-links/-/typedoc-plugin-djs-links-1.1.0.tgz>